eclair-core/src/test/scala/fr/acinq/eclair/payment/relay/NodeRelayerSpec.scala

```diff
@@ -323,6 +327,131 @@ class NodeRelayerSpec extends ScalaTestWithActorTestKit(ConfigFactory.load("appl
     register.expectNoMessage(100 millis)
   }
 
+  test("fail async payment when relay is not triggered") { f =>
+    val g = f.copy(nodeParams = f.nodeParams.copy(features = f.nodeParams.features.add(AsyncPaymentPrototype, Optional), relayParams = f.nodeParams.relayParams.copy(timeout = 5 seconds)))
```

Comment:
Change to using blockheight deltas instead of clocktime timeouts fixes this in [fcdc394](https://github.com/ACINQ/eclair/pull/2435/commits/fcdc394cb2751d7408318afb437cf49d18389d08)